### ISSUE TITLE
Broken settings GUI on Android 11 (OxygenOS V11.3)

### ISSUE THREAD

**Issue by MaxGyver83** [howeyc/crocgui#27]
I have installed the latest croc (1.10.1) via F-Droid on my _OnePlus Nord 2_. It works but the settings look broken:

![Screenshot_2022-01-31-21-10-02-73_0c31727280b74f78aaf91e6a8660278c](https://user-images.githubusercontent.com/13970628/151866004-1427cf47-1c48-45b3-97b4-502f089f22b0.jpg)

After tapping every single text field or checkbox, the content appears:

![Screenshot_2022-01-31-21-10-19-37_0c31727280b74f78aaf91e6a8660278c](https://user-images.githubusercontent.com/13970628/151866108-9cbcfcfa-46ba-4977-bd9e-3a7f1ac53cfe.jpg)

I don't know if this happens only on _OxygenOS_ but at least I can tell that I didn't see such a behavior in other apps.

**Comment by chipechop** [comment]
Same here, on Android 9: touching theme and then choosing "light", fixes the unreadable theme called "system".

![Screenshot_20220503-223254](https://user-images.githubusercontent.com/9009054/166561448-913e67b9-4ee6-4da2-a24b-25f336776ac5.jpg)
![Screenshot_20220503-223318](https://user-images.githubusercontent.com/9009054/166561459-32822b1b-26bf-4cc6-b914-215df4565ed2.jpg)

**Comment by MaxGyver83** [comment]
Meanwhile, I'm using the light theme, too.

**Comment by howeyc** [comment]
Interesting. Some kind of issue when first starts that objects aren't totally visible until interaction.

Is this something that you can reproduce? Does this happen every time you start the app?

**Comment by MaxGyver83** [comment]
Yes, it's still an issue. I changed the theme back to "system". Then, at first all settings stayed visible. But when I closed the app and started it again, all texts were gone again. When I change the theme to "light", all texts become visible immediately.

**Comment by chipechop** [comment]
> Yes, it's still an issue. I changed the theme back to "system". Then, at first all settings stayed visible. But when I closed the app and started it again, all texts were gone again. When I change the theme to "light", all texts become visible immediately.

Exactly the same happens, here

**Comment by howeyc** [comment]
If you get a chance, could you try this 
[test build](https://github.com/howeyc/crocgui/releases/download/v1.10.3/crocgui-test1.apk)

I can't reproduce myself and my attempted fix is just a guess. You'll need to uninstall the fdroid version to install this, as fdroid builds are signed using their key.

**Comment by chipechop** [comment]
Same: 
Missing contents in "system" theme, contents visible in "light" theme 

![Screenshot_20220505-184239](https://user-images.githubusercontent.com/9009054/166972130-876f6c74-ec33-4ea9-8a9c-8b7a6ccd1f71.jpg)
![Screenshot_20220505-184246](https://user-images.githubusercontent.com/9009054/166972135-5d08fb2b-2961-4392-bf0c-2f32b895f664.jpg)

**Comment by svenjacobs** [comment]
Same issue here on a Google Pixel 5 with Android 12 (build SQ3A.220705.003.A1). Croc version 1.10.4.
